SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -198,6 +291,8 @@ - (void)setupDatabaseWithSafeBlockingMigrations:(void (^_Nonnull)())safeBlocking
     [self.database registerExtension:[TSDatabaseSecondaryIndexes registerTimeStampIndex] withName:@"idx"];
     [OWSMessageReceiver syncRegisterDatabaseExtension:self.database];
 
+    setDatabaseInitialized();
```

Comment:
Right - I just want to capture what we believe in comments so when we revisit this later or get hit by a bus or something...